Makefile

```diff
@@ -111,4 +111,3 @@ endef
 
 SHELL = /bin/sh
 RAND = $(shell echo $$RANDOM)
-
```

Comment:
Revert this change and add `vertica` to the `DATABASE` variable at the top of this file